spec/controllers/admin/admin_custom_files_controller_spec.rb

```diff
@@ -35,7 +35,7 @@
         allow(SlideHub::Cloud::Engine::AWS).to receive(:upload_files).and_return(true)
         allow(SlideHub::Cloud::Engine::Azure).to receive(:upload_files).and_return(true)
         custom_file = create(:default_custom_file)
-        post 'create', params: { custom_file: { file: fixture_file_upload('dummy.txt', 'text/html'), description: 'hoge' } }
+        post 'create', params: { custom_file: { file: fixture_file_upload(Rails.root.join('spec', 'fixtures', 'dummy.txt'), 'text/html'), description: 'hoge' } }
```

Comment:
__[RuboCop]__

Metrics/LineLength: Line is too long. [161/160]


<!-- Comment by [SideCI](https://sideci.com) -->